.github/workflows/debian.yml

```diff
@@ -43,7 +44,7 @@ jobs:
         run: |
           sudo add-apt-repository ppa:coffeelibs/openjdk
           sudo apt-get update
-          sudo apt-get install debhelper devscripts dput coffeelibs-jdk-${{ env.COFFEELIBS_JDK_VERSION }} libgtk2.0-0
+          sudo apt-get install debhelper devscripts dput coffeelibs-jdk-${{ env.COFFEELIBS_JDK }}:amd64=${{ env.COFFEELIBS_JDK_VERSION }} libgtk2.0-0
```

Comment:
no need to specifiy an architecture here, this complicates porting this CI workflow to other archs.

Reply:
Fixed in https://github.com/cryptomator/cryptomator/pull/3163/commits/7594e51e2beb8f55ddb512c7ded4d62853b645ec